### DISCUSSION TITLE
A lora and model search function when you start typing

### DISCUSSION THREAD

**Discussion by sabi3293043** [comfyanonymous/ComfyUI#351] [Ideas]
A lora and model search function when you start typing or the ability to connect the prompt box into the input of the model or lora to manually enter its name.

the reason for me is i have 1574 loras and 50 models. and for the models its fine but for the loras i have to scroll forever to find what i am looking for

**Comment by KazimirIskander** [comment]
Not an answer to your question, but my current workaround for massive lists is to keep the ones I'm not using in a side directory and reloading the UI when I move a folder of LoRAs into the lora directory.
